getcontext/tracing/exceptions.py

```diff
@@ -0,0 +1,18 @@
+
+
+class ContextException(Exception):
+    pass
+
+
+class EvaluationException(ContextException):
```

Comment:
Small nit: my understanding is that exceptions in Python should normally end in "Error".

https://docs.python.org/3/tutorial/errors.html#user-defined-exceptions

Could we name these according to that pattern? I also think it'd be good to give them more precise names according to the semantics they represent, atm with EvaluationError vs EvaluationException they are v similarly named and it is hard for me to keep in my head which one is for which outcome.

Perhaps EvaluationsFailedError for when the evaluations don't pass and something like `InternalContextError` for when running evaluations produces an error?

Reply:
Sounds good, regarding exception vs error, I don't think it is a strong requirement, your suggestions look good so will implement